packages/block-serialization-default-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/block-serialization-default-parser",
-	"version": "1.0.0-rc.0",
+	"version": "1.0.1",
```

Comment:
I think the idea is that we don't touch the package.json ourselves, we let lerna do so at the moment of releasing. What we need to do though is to just add a not in the changelog and add an `( unreleased )` next to the version, that way the person that will publish the packages to npm will know what version to pick using lerna. 

you can check CHANGELOG files from other packages

Reply:
thanks @youknowriad! I had no idea but TIL